cypress/support/e2e.js

```diff
@@ -0,0 +1,22 @@
+
+// ***********************************************************
+// This example support/e2e.js is processed and
+// loaded automatically before your test files.
+//
+// This is a great place to put global configuration and
+// behavior that modifies Cypress.
+//
+// You can change the location of this file or turn off
+// automatically serving support files with the
+// 'supportFile' configuration option.
+//
+// You can read more here:
+// https://on.cypress.io/configuration
+// ***********************************************************
+
+// Import commands.js using ES2015 syntax:
+import './commands'
+import 'cypress-plugin-api'
+import 'cypress-axe'
+// Alternatively you can use CommonJS syntax:
+// require('./commands')
```

Comment:
```suggestion
// require('./commands')

```

É uma convenção deixar uma linha em branco ao final de cada arquivo `.js`.

Análise estática de código com [ESLint](https://eslint.org/) poderia ajudar nisso, tornando tal verificação automática.